### DISCUSSION TITLE
Sample MRF Files

### DISCUSSION THREAD

**Discussion by jayfeins** [CMSgov/price-transparency-guide#318] [Q&A] [answered]
Would it be possible for CMS to provide sample MRFs? (Files containing sample/dummy values)

**Comment by shaselton-usds** [comment, marked as answer]
@jayfeins Have you had a chance to look into the examples directory [here](https://github.com/CMSgov/price-transparency-guide/tree/master/examples)

**Reply by Kvillafor** [reply]
Can you link me to the example for the Allowed Amounts multiple plan reporting example? I can only find one for the Negotiated Rates file.

**Reply by shaselton-usds** [reply]
Here in the table of contents, [multiple plans are listed](https://github.com/CMSgov/price-transparency-guide/blob/master/examples/table-of-contents/table-of-contents-sample.json#L5-L15) for a [single allowed amount file](https://github.com/CMSgov/price-transparency-guide/blob/master/examples/table-of-contents/table-of-contents-sample.json#L23-L26).

I think this was what you were asking. Please let me know if it wasn't.
